imports/ui/QuestionSidebar.jsx

```diff
@@ -51,7 +53,11 @@ export class QuestionSidebar extends ControlledForm {
       })
       this.forceUpdate()
     })
-    
+
+    Meteor.call('courses.courseRequiresApprovedQuestions',this.props.courseId, (e, approved) => {
```

Comment:
Rename the method to courses.publicQuestionsRequireApproval, so that if the user is looking at public questions in a course that does not require public questions to be approved, then the user can show only approved questions.
That's the point of this state variable, right?